src/EnergyPlus/SystemAvailabilityManager.hh

```diff
@@ -541,19 +518,19 @@ struct SystemAvailabilityManagerData : BaseGlobalStruct
     Array1D<Real64> OptStart_AdaTempGradTrdHeat; // Heating temp gradient for previous days - used in CalcOptStartSysAvailMgr
     Array1D<Real64> OptStart_AdaTempGradTrdCool; // Cooling temp gradient for previous days - used in CalcOptStartSysAvailMgr
 
-    EPVector<SystemAvailabilityManager::DefineSchedSysAvailManager> SchedSysAvailMgrData;
-    EPVector<SystemAvailabilityManager::DefineSchedOnSysAvailManager> SchedOnSysAvailMgrData;
-    EPVector<SystemAvailabilityManager::DefineSchedOffSysAvailManager> SchedOffSysAvailMgrData;
-    EPVector<SystemAvailabilityManager::DefineNightCycSysAvailManager> NCycSysAvailMgrData;
-    EPVector<SystemAvailabilityManager::DefineDiffTSysAvailManager> DiffTSysAvailMgrData;
-    EPVector<SystemAvailabilityManager::DefineHiLoSysAvailManager> HiTurnOffSysAvailMgrData;
-    EPVector<SystemAvailabilityManager::DefineHiLoSysAvailManager> HiTurnOnSysAvailMgrData;
-    EPVector<SystemAvailabilityManager::DefineHiLoSysAvailManager> LoTurnOffSysAvailMgrData;
-    EPVector<SystemAvailabilityManager::DefineHiLoSysAvailManager> LoTurnOnSysAvailMgrData;
-    EPVector<SystemAvailabilityManager::DefineNightVentSysAvailManager> NVentSysAvailMgrData;
-    EPVector<SystemAvailabilityManager::DefineHybridVentSysAvailManager> HybridVentSysAvailMgrData;
-    EPVector<SystemAvailabilityManager::SysAvailManagerList> SysAvailMgrListData;
-    EPVector<SystemAvailabilityManager::DefineOptStartSysAvailManager> OptStartSysAvailMgrData;
+    EPVector<SystemAvailabilityManager::Scheduled> SchedData;
+    EPVector<SystemAvailabilityManager::ScheduledOn> SchedOnData;
+    EPVector<SystemAvailabilityManager::ScheduledOff> SchedOffData;
+    EPVector<SystemAvailabilityManager::NightCycle> NightCycleData;
+    EPVector<SystemAvailabilityManager::DiffThermo> DiffThermoData;
+    EPVector<SystemAvailabilityManager::HiLoTemp> HiTurnOffData;
+    EPVector<SystemAvailabilityManager::HiLoTemp> HiTurnOnData;
+    EPVector<SystemAvailabilityManager::HiLoTemp> LoTurnOffData;
+    EPVector<SystemAvailabilityManager::HiLoTemp> LoTurnOnData;
+    EPVector<SystemAvailabilityManager::NightVent> NightVentData;
+    EPVector<SystemAvailabilityManager::HybridVent> HybridVentData;
+    EPVector<SystemAvailabilityManager::List> ListData;
+    EPVector<SystemAvailabilityManager::OptimumStart> OptimumStartData;
```

Comment:
Happy to rename if these EPVector names look weird. 

“There are only two hard things in computer programming: cache invalidation and naming things.” - Phil Karlton